css-will-change-transform-rasterization/demo.js

```diff
@@ -0,0 +1,9 @@
+window.addEventListener('load', function() {
+  requestAnimationFrame(function() {
```

Comment:
I *think* this ensures that rasterization has happened (otherwise we might not see the blurriness)